test_jsonify_stored/hooks.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import SUPERUSER_ID, api
+
+
+def pre_init_hook(cr):
+    """The test model depends on model_export, so we need it before model init."""
+    env = api.Environment(cr, SUPERUSER_ID, dict())
+    export = env.ref("test_jsonify_stored.model_export", raise_if_not_found=False)
```

Comment:
why do you need this? The ir.export record should be created explicitly by the implementing module.

Reply:
Because when the models are setup, the `@api.depends(lambda self: self._jsonify_get_export_depends())` on the implementing model would crash, as the export does not exist yet.

Reply:
tnx for the explanation. For everyone else and your future self: mention this in a comment :wink: 

Nevertheless: can't we do this in the mixin? I mean, this is a mandatory piece. It must be there. Period.
And I can imagine a lot of headaches of ppl not adding this init hook or missing that record in many other ways.

What if `_jsonify_get_export` creates it if missing. And what if we define a computed name/xid for it?
Something like `{module}.jsonify_export_{model._table}` maybe?
It would also help in keeping ir.exports records name and xid consistent across modules.

Reply:
@len-foss ping :)